src/Sylius/Bundle/CoreBundle/Form/Type/CustomerGuestType.php

```diff
@@ -9,10 +9,9 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Bundle\UserBundle\Form\Type;
+namespace Sylius\Bundle\CoreBundle\Form\Type;
```

Comment:
We discussed yesterday, that CustomerGuestType is only for Sylius shop implementation and shouldn't exist in CustomerBundle.